lib/rubocop/cop/style/file_dirname_level.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for repeated `File.dirname`.
+      # It replaces repeats with the level argument introduced in Ruby 3.1.
+      #
+      # @example
+      #
+      #   # bad
+      #   File.dirname(File.dirname(path))
+      #
+      #   # good
+      #   File.dirname(path, 2)
+      #
+      class FileDirnameLevel < Base
```

Comment:
I wonder if `NestedFileDirname` is not a better name, but I'm fine with the proposed name.

Reply:
Yeah, I was wondering whether to use that name. So, I've updated to `NestedFileDirname` to be consistent with the doc.